### ISSUE TITLE
Could not pass event PlayerInteractEvent to CratesPlus

### ISSUE THREAD

**Issue by JoshMerlino** [ConnorLinfoot/CratesPlus#776]
Whenever a player opens a chest that is not registered as a crate, the following error is printed to the console. Also upon logging, The plugin breaks (Holograms above crates disappear and crates become normal chests). They however work fine after the plugin is reloaded until another non crate chest is opened by a player.

### Plugin Configuration
All plugin configuration can be found [here](https://gist.github.com/JoshMerlino/6bc18a364ff085a76c37e23aae8a9937).

### Server Info
Server IP: `mayhemmc.uk.to`

Version: git-Paper-378 (MC: 1.15.2) (Implementing API version 1.15.2-R0.1-SNAPSHOT)

Plugins (60): AutoMessage, BigAuction*, BossGamble, BuycraftX, ChatItem*, ChatReaction, ClearLag, CombatLogX, CombineDeluxe, CompatNoCheatPlus*, CratesPlus, CrazyEnvoy, CubCustomDrops, dynmap*, EnchantPlus, EnchLimiter, Essentials, EssentialsChat, EssentialsSpawn, Factions, GenBucket, HeadHunter, HolographicDisplays, ItemInChatFree, LuckPerms, mcMMO, MMCWebHook, MobStacker*, Multiverse-Core, NoBabyZombies*, NoCheatPlus, NoNetherRoof, NoPhantoms, ObsidianDestroyer, OldCombatMechanics, OwnGarden, PlaceholderAPI, PlayerVaults, PlugMan, ProtocolLib, RandomSpawnPlus, SavageFTOP, SellChest, ShopGUIPlus, ShopGUIPlusSilkSpawnersBridge, SilkSpawners, SimpleRename, SuperbVote, SuperVanish, TabList, Trade, Vault, ViaBackwards, ViaVersion, ViewDistanceTweaks, VoidGenerator, Votifier, WorldEdit, WorldGuard, Zenchantments

### Link/paste of stack trace/error (if any)
```
[22:41:44 ERROR]: Could not pass event PlayerInteractEvent to CratesPlus v4.5.2
java.lang.NullPointerException: null
        at plus.crates.Listeners.PlayerInteract.onPlayerInteract(PlayerInteract.java:45) ~[?:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor714.execute(Unknown Source) ~[?:?]
        at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.15.2.jar:git-Paper-378]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.15.2.jar:git-Paper-378]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.15.2.jar:git-Paper-378]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:607) ~[patched_1.15.2.jar:git-Paper-378]
        at org.bukkit.craftbukkit.v1_15_R1.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:471) ~[patched_1.15.2.jar:git-Paper-378]
        at net.minecraft.server.v1_15_R1.PlayerInteractManager.a(PlayerInteractManager.java:456) ~[patched_1.15.2.jar:git-Paper-378]
        at net.minecraft.server.v1_15_R1.PlayerConnection.a(PlayerConnection.java:1419) ~[patched_1.15.2.jar:git-Paper-378]
        at net.minecraft.server.v1_15_R1.PacketPlayInUseItem.a(PacketPlayInUseItem.java:27) ~[patched_1.15.2.jar:git-Paper-378]
        at net.minecraft.server.v1_15_R1.PacketPlayInUseItem.a(PacketPlayInUseItem.java:5) ~[patched_1.15.2.jar:git-Paper-378]
        at net.minecraft.server.v1_15_R1.PlayerConnectionUtils.lambda$ensureMainThread$1(PlayerConnectionUtils.java:23) ~[patched_1.15.2.jar:git-Paper-378]
        at net.minecraft.server.v1_15_R1.TickTask.run(SourceFile:18) ~[patched_1.15.2.jar:git-Paper-378]
        at net.minecraft.server.v1_15_R1.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:136) ~[patched_1.15.2.jar:git-Paper-378]
        at net.minecraft.server.v1_15_R1.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) ~[patched_1.15.2.jar:git-Paper-378]
        at net.minecraft.server.v1_15_R1.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:109) ~[patched_1.15.2.jar:git-Paper-378]
        at net.minecraft.server.v1_15_R1.MinecraftServer.ba(MinecraftServer.java:1105) ~[patched_1.15.2.jar:git-Paper-378]
        at net.minecraft.server.v1_15_R1.MinecraftServer.executeNext(MinecraftServer.java:1098) ~[patched_1.15.2.jar:git-Paper-378]
        at net.minecraft.server.v1_15_R1.IAsyncTaskHandler.executeAll(IAsyncTaskHandler.java:95) ~[patched_1.15.2.jar:git-Paper-378]
        at net.minecraft.server.v1_15_R1.MinecraftServer.a(MinecraftServer.java:1232) ~[patched_1.15.2.jar:git-Paper-378]
        at net.minecraft.server.v1_15_R1.MinecraftServer.run(MinecraftServer.java:970) ~[patched_1.15.2.jar:git-Paper-378]
        at java.lang.Thread.run(Thread.java:834) [?:?]
```

### Actions to reproduce (if known):
Open a chest that is not a crate.

**Comment by ItzCreeperBoy27** [comment]
http://www.mediafire.com/file/u8o4pnrwknf6547/Crates.jar/file heres a working cratesplus that fixes that error

**Comment by ConnorLinfoot** [comment]
Fixed in upcoming update #793
